vortex-api/src/main/java/mil/army/usace/hec/vortex/io/DataWriter.java

```diff
@@ -91,6 +91,12 @@ public DataWriter build() {
                 return new NetcdfDataWriter(this);
             }
 
+            //PathMatcher hdf5Matcher = FileSystems.getDefault().getPathMatcher("glob:**/*.p**.{h5,hdf5}"); //trying to match a RAS plan output e.g. MyModel.p01.hdf
```

Comment:
Can you get the RAS "p" file path matching working? I think the hdf pattern matching will cause problems in the future if we ever implement a generalized HDF writer.

Reply:
fixed.